logp/core.go

```diff
@@ -236,6 +236,34 @@ func makeEventLogOutput(cfg Config, enab zapcore.LevelEnabler) (zapcore.Core, er
 	return wrappedCore(core), nil
 }
 
+// WithFileOrStderrOutput configures the logger with an output based on cfg.
+// If neither `cfg.ToFiles` nor `cfg.ToSdterr` are true, then a noop output
+// is created. If both are true, the file output has preference.
+func WithFileOrStderrOutput(cfg Config) func(zapcore.Core) zapcore.Core {
+	var out zapcore.Core
+	var err error
+
+	switch {
+	case cfg.ToFiles:
+		out, err = makeFileOutput(cfg, cfg.Level.ZapLevel())
+	case cfg.ToStderr:
+		out, err = makeStderrOutput(cfg, cfg.Level.ZapLevel())
```

Comment:
If both ToFiles and ToStderr are true we prefer ToFiles?

Reply:
Yes, that's the indented behaviour, because it's sensitive data and when running in a container environment we log to stdout/stderr, I opted to have the file output having priority to avoid leaking event data by accident/missconfiguraiton.

Reply:
Done: 24878e6